moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1925,7 +1925,7 @@ double RobotState::computeCartesianPath(const JointModelGroup* group, std::vecto
     Eigen::Affine3d pose(start_quaternion.slerp(percentage, target_quaternion));
     pose.translation() = percentage * rotated_target.translation() + (1 - percentage) * start_pose.translation();
 
-    if (setFromIK(group, pose, link->getName(), 1, 0.0, validCallback, options))
+    if (setFromIK(group, pose, link->getName(), 0, 0.0, validCallback, options))
```

Comment:
I think, a single attempt is used on purpose here: In a Cartesian path, it doesn't make sense to use a random seed for IK. This essentially enforces non-continuous joint-space motions.

Reply:
hrm, good point. I will drop this change, and add this as a comment.